d2bs/kolbot/libs/common/Loader.js

```diff
@@ -105,7 +106,7 @@ var Loader = {
 								reconfiguration = typeof Scripts[i] === 'object';
 
 								if (typeof (global[i]) === "function") {
-									if (i !== "Test" && i !== "Follower") {
+									if (i !== "Test" && i !== "Follower" && i !== "Duriel") {
```

Comment:
I think change this to something dynamic, like have a array of scripts to skip, initialize with Test and Follower
then in Summoner.js, insert Duriel if book is used
something like `Loader.townCheck.push("Duriel");`